packages/core/echo/echo-db/src/client/doc-handle-client.ts

```diff
@@ -0,0 +1,172 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import { EventEmitter } from 'eventemitter3';
```

Comment:
should we replace our `Event` in `@dxos/async` with this?

Reply:
please no. IMO EventEmitter is the most dangerous, most abused class in the node ecosystem. Event is much more targetted, typed, etc.

Reply:
Agree with Rich here, it is done to match API of automerge repo, but we will move from it soon